docs/blog/rendered/ibis-version-6.0.0-release.ipynb

```diff
@@ -0,0 +1,1092 @@
+{
```

Comment:
<p>This can be changed to <code>from ibis import udf</code>.  <span class="ql-cursor">﻿</span></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ibis-project/ibis/pull/6552/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>